src/lib/djerba/helpers/pwgs_cardea_helper/helper.py

```diff
@@ -60,11 +60,19 @@ def get_cardea(self, requisition_id, cardea_url):
                 if qc_group['libraryDesignCode'] == "PG":
                     group_id = qc_group['groupId']
             
-            if len(projects) > 1 or len(projects) < 1:
-                msg = "{0} case(s) were found. Only 1 case is expected".format(len(cases))
+            if len(projects) < 1:
+                msg = "No projects were found. 1 project in the 'Accredited' or 'Accredited with Clinical Report' pipeline is required"
                 raise ValueError(msg)
             else:
-                project_id = projects[0]['name']
+                clinical = "no"
+                for project in projects:
+                    if "Accredited" in project["pipeline"]:
+                        project_id = project['name']
+                        clinical = "yes"
+                if clinical == "no":
+                    print(project['pipeline'])
+                    msg = "No projects in the 'Accredited' or 'Accredited with Clinical Report' pipeline were found; 1 is required"
```

Comment:
Please write this message to the log with `self.logger.error`. Also, could you use Boolean `True` and `False` instead of strings `"yes"` and `"no"` here?